### ISSUE TITLE
Column width cannot be changed

### ISSUE THREAD

**Issue by AEgit** [JabRef/jabref#10516]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

Windows

### Details on version and operating system

Windows 10

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

JabRef 5.11--2023-10-17--8ede31a
Windows 10 10.0 amd64 
Java 21-internal 
JavaFX 20.0.2+3

1. Open a JabRef database
2. Try to modify the column widths in the main table view by trying to drag'n'drop the respective column boundaries
3. Nothing happens - you cannot change the column width

This is the main issue. The following observations appear linked, but are probably less problematic for the user (?).

Additional observations:
4. If you resize then your JabRef window (e.g., make it take up less space on your desktop), the column widths will resize as well
5. But if you bring JabRef back to its original (larger) window size, the column widths will stay resized to the level they were at step (4)
6. Given that you face the issue described in steps (1)-(3) you cannot manually change the column sizes yourself

Furthermore note:
7. If you swap in main table view to another JabRef database and then to the one you had been working before, the column widths are again resized to what they were at step (3)

**Comment by Siedlerchr** [comment]
I think I know where your problem is:
Disable  "Fit table horizontally on the screen" (Default value is of in newer versions but was on in earlier)
<img width="868" alt="grafik" src="https://github.com/JabRef/jabref/assets/320228/14fb0474-7e23-4743-b4ee-2de5011c7308">

**Comment by AEgit** [comment]
Cheers, and sorry for the unnecessary bug report. I hadn't checked the recent development in the dev version in a while and had missed this change. Thanks for quickly pointing me into the right direction!

As for the steps (4) to (7): Is that the intended behaviour with "Fit table horizontally on the screen" turned on? If yes, then this issue can be closed.

**Comment by Siedlerchr** [comment]
Yes, as the name implies "Fit table horizontally on the screen" will automatically calculate the column size based on the window size
Not sure about step 7, but I think this is a consequence that when you "reactivate" the other library, the size calculations triggered again.

However, as this Feature is toggled off by default I would not spend too much time thinking about this

**Comment by ThiloteE** [comment]
Only resetting the preferences will give you the "real" new experience of newer versions of JabRef.

**Comment by AEgit** [comment]
Cheers and thanks again for the quick help!

**Comment by koppor** [comment]
Note that we wanted to add a better heuristics at https://github.com/JabRef/jabref/pull/7181.

Maybe it's time to work on this again, because there are also updates at JavaFX -- https://bugs.openjdk.org/browse/JDK-8293119

**Comment by tobiasdiez** [comment]
This was definitely working at some point even with "fit table horizontally on screen" activated. Thus reopening.
